### DISCUSSION TITLE
Can `actions-user` user be excluded from recent activity commits?

### DISCUSSION THREAD

**Discussion by andry81** [lowlighter/metrics#833] [Q&A] [answered]
This is default github action bot-user which can impersonate a commit user, which brings spam in the activity list.

**Comment by lowlighter** [comment, marked as answer]
Hi 👋 !

Yes, you can use `plugin_activity_ignored` option. Similar options exists for other plugins.

Note that each "ignored" option also inherits from `users_ignored` option which is set by default to `github-actions[bot], dependabot[bot], dependabot-preview[bot]`

**Reply by andry81** [reply]
What about the default link: `https://metrics.lecoq.io/about/<user>` ? How to exclude it from the link?

**Reply by lowlighter** [reply]
Everything in `/about` is not configurable by design, but since it's still use default values it may actually already ignore susmentionned bots

**Reply by andry81** [reply]
It does not ignore bot commits.

**Reply by lowlighter** [reply]
I assume it's because the bot you're referring to is [`actions-user`](https://github.com/actions-user) not `github-actions[bot]`

"Real bots" accounts have `[bot]` in their username and does not actually have any user profile page. 
Additionally, AFAIK they also have a square icon instead of a round one:
![image](https://user-images.githubusercontent.com/22963968/152897005-3d9a74db-1d55-4b71-9201-b06fd34ea8b1.png) ![image](https://user-images.githubusercontent.com/22963968/152897614-b341c5ab-ca3e-43ec-b6a4-601f6bce549b.png) ![image](https://user-images.githubusercontent.com/22963968/152897634-3e479373-d563-4cc9-ab1c-217df1362d2f.png)

Where as `action-user` is actually a user account:
![image](https://user-images.githubusercontent.com/22963968/152897446-c37a4547-4c81-43a5-9a07-d59f48ac515a.png)

**Reply by andry81** [reply]
It is not a user, that is a bot which account used as for commits from GitHub action pipeline. And does not matter what icons and stickers is does use.

**Reply by lowlighter** [reply]
The `actions-user` has been added to the list of ignored users since 👍 
(c.f. [`users_ignored`](https://github.com/lowlighter/metrics/tree/master/source/plugins/base#users_ignored) option)

**Reply by andry81** [reply]
Seems `https://metrics.lecoq.io/about/<user>` still shows `actions-user` commits.

**Reply by lowlighter** [reply]
Indeed, this will be patched by #890 👍 

The reason is that despite being committed by the bot, you're still the actor on it so that's why it's not properly ignored (other events should be hidden though).

While the github login is not available in the commits payload (since it's git), above pr will support ignoring commits related stuff by providing emails (added the respective email for `actions-user`) so on next version you'll get your feed without bots actions 

Thanks for your feedback 👍
